.github/workflows/test.yml

```diff
@@ -2,11 +2,14 @@ name: Test
 
 on:
   push:
+  pull_request:
+    branches:
+      - master
 
 jobs:
   inch:
     name: Analyse Documentation
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
👍 As Ubuntu's 18.04 version is a LTS release and has support until April 2023, this is reasonable to lock to for now. We'll handle the update from 18.04 to 20.04 (the latest LTS release) separately.